source/sass/main.scss

```diff
@@ -325,13 +327,17 @@ button {
     .container {
       margin-top: 0;
       margin-bottom: 0;
-      display: flex;
-      height: 220px;
-      justify-content: space-around;
-      align-items: center;
+      min-height: 220px;
+
+      @media screen and (min-width: $bp-md) {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+      }
     }
 
     .info-box {
```

Comment:
Actually, those margins should be overlapping. However, `info-box` is erroneously `display: inline-block;` which isn't actually being used for anything, and causing the margins to not overlap. If it's `display: block;` and properly cleared and centered, we can get the same result.